app/build.gradle

```diff
@@ -7,6 +7,7 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.facebook.testing.screenshot'
 
 def properties = new Properties()
+properties.putAll(System.getenv())
```

Comment:
We put all the system env properties to the dictionary by default to have access to CI env tokens.